### DISCUSSION TITLE
Generating create table query from struct

### DISCUSSION THREAD

**Discussion by barsdeveloper** [rbatis/rbatis#556] [Q&A]
Hello, I'm new to rbatis but I was wondering is there something similar to schema.create_table_from_entity(entity_object) from Sea ORM that gives a query that when executed creates a table suitable for a Entity. With Sea ORM I was using a code first approach that on the first run sets up the database to be coherent with the entities I have. Is there something similar in rbatis?

**Comment by zhuxiujia** [comment]
No, as a semi-automatic ORM, rbatis can specify table names or use custom interceptors to force changes to table names，you can [see](https://github.com/rbatis/rbatis/blob/master/example/src/crud.rs)  of
`crud!(Activity {},"activity");`
